src/web/src/components/LandingButtons.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from 'next/link';
+import { makeStyles } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import useAuth from '../hooks/use-auth';
+
+const useStyles = makeStyles((theme) => ({
+  buttonsContainer: {
+    position: 'absolute',
+    color: 'white',
+    top: '20px',
+    width: '250px',
+    right: '5%',
+    display: 'flex',
+    justifyContent: 'space-around',
+    '& > .MuiButton-root': {
+      color: 'white',
+      border: '1.5px solid white',
+      '&:hover': {
+        color: '#9ABDFF',
```

Comment:
Let's add this colour to the theme

Reply:
I don't want to connect the landing page with the theme colors now, let's make this when implementing a dark/light themes in the landing page. For now, let's land the new land page without a dark/light theme.